webapp/app/components/beam-files/component.js

```diff
@@ -62,6 +65,11 @@ export default Component.extend({
   actions: {
     filterChange: function(newFilter) {
       this.set("fileFilter", newFilter);
+    },
+    sortBy: function(property, acsending) {
+      var method = (acsending) ? 'asc': 'desc';
+      this.get('sortKeys').pop()
```

Comment:
I think that you don't have to use `this.get` in recent versions of Ember. See if `this.sortKeys.pop()` works.